schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED  ALWAYS AS IDENTITY,
+    name CHAR NOT NULL,
```

Comment:
- [ ] This command means that your name column will not be null and wi only have one character long, however, the name should not only be one character long. Kindly make sure to make the name at least < 20 characters long. example: `name CHAR(25) NOT NULL,`

here is the error I am facing :point_down: 

![image](https://user-images.githubusercontent.com/65068771/155698384-45fd9c1b-f385-4e23-a3a9-d81fd2220840.png)